src/App/deployments/firebase.js

```diff
@@ -1,31 +1,42 @@
-// TODO @brown-ccv #183: Upgrade to modular SDK instead of compat
-import firebase from "firebase/compat/app";
-import "firebase/compat/firestore";
+import { initializeApp } from "firebase/app";
+import {
+  getFirestore,
+  connectFirestoreEmulator,
+  doc,
+  setDoc,
+  addDoc,
+  collection,
+} from "firebase/firestore";
 
 // Initialize Firebase and Firestore
-firebase.initializeApp({
+const app = initializeApp({
   apiKey: import.meta.env.VITE_API_KEY,
   authDomain: import.meta.env.VITE_AUTH_DOMAIN,
   projectId: import.meta.env.VITE_PROJECT_ID ?? "no-firebase",
   storageBucket: import.meta.env.VITE_STORAGE_BUCKET,
   messagingSenderId: import.meta.env.VITE_MESSAGING_SENDER_ID,
   appId: import.meta.env.VITE_APP_ID,
 });
-export const db = firebase.firestore();
+export const db = getFirestore(app);
```

Comment:
Can you rename these to be all caps? So `APP` and `DB`

Reply:
Should be changed to `APP` and `DB`! I'll go ahead and merge it.